### PR TITLE
Drop IE7 support in the grid framework

### PR DESCRIPTION
**Pull request by BPScott**
Moving to use box-sizing:border-box means we can remove that inner div
that we previously needed to create fixed width gutters.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Core/Gel/Objects/_grid.scss [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
  *
- * An IE 7 compatible implementation of a fluid grid with fixed gutter sizes.
+ * An IE 8 compatible implementation of a fluid grid with fixed gutter sizes.
  * Use in conjunction with the width objects to set the sizing of your grid
  * items.
  *
- * The .grid__inner div is required for IE7 compatibility. If we drop support
- * for IE 7 then we can use box-sizing:border-box; and apply the padding
- * directly to the .grid element, thus removing the need for the nested div.
+ * The original grid from programmes had IE7 compatibility by using an inner
```
Comment (votemike) [inline src/Ds2013/Core/Gel/Objects/_grid.scss]: Is this comment useful?

File: src/Ds2013/Core/Gel/Objects/_widths.scss [hunk 2]
```diff
@@ -1,68 +1,77 @@
 /**
  * Widths.
  *
- * Human readable width abstractions.
+ * In the old system we used word based names for the classes e.g. "one-third"
```
Comment (votemike) [inline src/Ds2013/Core/Gel/Objects/_widths.scss]: Are comments about how a system used to work useful?

File: src/Ds2013/Core/Gel/Objects/_grid.scss (hunk 1 above)
Comment (BPScott) [inline src/Ds2013/Core/Gel/Objects/_grid.scss]: I was thinking that in the move most of the the CSS classes shall be a copy+paste, however as this bit is deviating from what already exists it'd be worth pointing that divergence out to save people thinking "I copy+pasted something that mentions grid__inner but it doesn't do anything".

Comment (votemike) [inline src/Ds2013/Core/Gel/Objects/_grid.scss]: Fair enough

**Pull request merged**